.github/workflows/precompiled.yml

```
@@ -0,0 +1,350 @@
+          path: gems
+          retention-days: 7
+
+  cruby-linux-install:
+    needs: ["cruby-package"]
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby: ["2.7", "3.0", "3.1", "3.2"]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: "${{matrix.ruby}}"
+      - uses: actions/download-artifact@v3
+        with:
+          name: cruby-gem
+          path: gems
+      - run: ./scripts/test-gem-install gems
+
+  cruby-osx-install:
+    needs: ["cruby-package"]
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby: ["3.2"]
+        sys: ["enable", "disable"]
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: "${{matrix.ruby}}"
+      - uses: actions/download-artifact@v3
+        with:
+          name: cruby-gem
+          path: gems
+      - run: ./scripts/test-gem-install gems
+
+  cruby-windows-install:
+    needs: ["cruby-package"]
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby: ["2.7", "3.0"]
+        sys: ["enable", "disable"]
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: "${{matrix.ruby}}"
+      - uses: actions/download-artifact@v3
+        with:
+          name: cruby-gem
+          path: gems
+      - run: ./scripts/test-gem-install gems
+        shell: bash
+
+  cruby-windows-install-ucrt:
+    needs: ["cruby-package"]
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby: ["3.1", "3.2"]
+        sys: ["enable", "disable"]
+    runs-on: windows-2022
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ruby/setup-ruby@v1
+        with:
+         ruby-version: "${{matrix.ruby}}"
+      - uses: actions/download-artifact@v3
+        with:
+          name: cruby-gem
+          path: gems
+      - run: ./scripts/test-gem-install gems
+        shell: bash
+
+  cruby-native-package:
+    needs: ["rcd_image_version"]
+    strategy:
+      fail-fast: false
+      matrix:
+        plat:
+          - "aarch64-linux"
+          - "arm-linux"
+          - "arm64-darwin" # github actions does not support this runtime as of 2022-12, but let's build anyway
+          - "x64-mingw-ucrt"
+          - "x64-mingw32"
+          - "x86-linux"
+          - "x86-mingw32" # github actions does not support this runtime as of 2022-12, but let's build anyway
+          - "x86_64-darwin"
+          - "x86_64-linux"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/cache@v3
+        with:
+          path: ports/archives
+          key: archives-ubuntu-${{hashFiles('ext/re2/extconf.rb')}}
+      - env:
+          DOCKER_IMAGE: "ghcr.io/rake-compiler/rake-compiler-dock-image:${{needs.rcd_image_version.outputs.rcd_image_version}}-mri-${{matrix.plat}}"
```

Rather than doing this dynamically, given we know the version of Rake Compiler Dock we have, could we hard-code it to 1.3.0 instead?

```suggestion
          DOCKER_IMAGE: "ghcr.io/rake-compiler/rake-compiler-dock-image:1.3.0-mri-${{matrix.plat}}"
```

---

Yes, that would be fine.